common/src/main/java/com/github/streamshub/console/config/Named.java

```diff
@@ -0,0 +1,16 @@
+package com.github.streamshub.console.config;
+
+import java.util.Collection;
+
+public interface Named {
+
+    static boolean uniqueNames(Collection<? extends Named> configs) {
```

Comment:
Should we use a more generic variable than `configs`? Or change the interface to be `NamedConfig`

Reply:
Latest commit renames the arg from `configs` to `items`.